streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractKeyValueStoreTest.java

```diff
@@ -188,7 +188,55 @@ public void testPutGetRange() {
     }
 
     @Test
-    public void testPutGetRangeWithDefaultSerdes() {
+    public void testPutGetReverseRange() {
```

Comment:
I think we should add some tests to `RocksDBTimestampedStoreTest`. I thought it would extend `AbstractKeyValueStoreTest` and thus benefit from everything you added here, but doesn't seem to be the case :/ 
Personally I found the `RocksDBDualCFIterator` logic a bit difficult to follow even before the reverse iteration,  so it would be nice to have some tests specifically covering reverse iterators over multi-column-family timestamped stores

Reply:
Just realized that, I also thought that path was tested. Good catch!

Reply:
@ableegoldman I have extended `RocksDBTimestampedStoreTest` to use `reverseAll` and `reverseRange` as part of the current tests. 
Unfortunately, `AbstractKeyValueStoreTest` tests do not fit with the creation path of Timestamped stores as pre inserted data is required. 
Will add this to the same JIRA ticket to consider when refactoring iterators and tests.

Reply:
Ah, that makes sense I guess. Looks like the new additions to `RocksDBTimestampedStoreTest` cover the cross-column family code path so that's good enough for now